README.md

```diff
@@ -276,10 +280,9 @@ The `@WithSpan` annotation requires code changes to implement. You can
 disable the annotation at runtime via the exclude configuration or
 environment variables:
 
-| System property                  | Environment variable             | Purpose                                                              |
-|----------------------------------|----------------------------------|----------------------------------------------------------------------|
-| trace.classes.exclude            | TRACE_CLASSES_EXCLUDE            | Exclude classes with the `@WithSpan` annotation                      |
```

Comment:
Isn't this property still there?

Reply:
I moved documentation of that property to the "Suppressing specific auto-instrumentation" section above (which links to a new doc page)

Reply:
Ah sorry missed that markdown in the middle of the java files!